runtime/compiler/ilgen/IlGenerator.cpp

```diff
@@ -42,6 +42,20 @@
 
 #define OPT_DETAILS "O^O ILGEN: "
 
+#define JSR292_MethodHandle   "java/lang/invoke/MethodHandle"
```

Comment:
These defines are duplicating defines in j9method.cpp I think we should have a header for the JSR292 defines and include the header in the files where we want the defines to apply so that the definitions are just in once place.